hera_pspec/pspecdata.py

```diff
@@ -3165,6 +3888,7 @@ def trim_dset_lsts(self, lst_tol=6):
             trim_inds = np.array([l not in common_lsts for l in lst_arrs[i]])
             if np.any(trim_inds):
                 self.dsets[i].select(times=dset.time_array[~trim_inds])
+        self.Ntimes = self.dsets[i].Ntimes#update Ntimes Dammnit!
```

Comment:
The comment is not needed. I have removed it. 

The line does need to exist however because previously, trimming the lsts did not update the number of times in the pspecdata object, leading to mismatches in instantiated data arrays (which use Ntimes) and the attached data sets. 

the unittest for trim_lsts did not test that Ntimes had been properly updated. I've updated it to ensure this as well.